### ISSUE TITLE
Docs: Add Reddit to AMO-list (addons.mozilla.org)

### ISSUE THREAD

**Issue by Jonta** [SimonBrazell/privacy-redirect#205]
Currently not mentioned in the bulleted list there: https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/

Something to add to a checklist, perhaps. =)

**Comment by SimonBrazell** [comment]
Yeah sure will do. Can also add Simply Translate and search engines to the list now while I'm at it 🙂

**Comment by Jonta** [comment]
What's blocking this? =)
